source/tutorials/kubernetes.rst

```diff
@@ -3,354 +3,761 @@
 
-Cloud native setup automation
-*****************************
+#. Start the Docker service and enable it to start automatically at boot:
 
-Optional: Clone the `cloud-native-setup`_ repository on your system and follow the instructions. This repository includes helper scripts to automate configuration.
+   .. code::
 
-Package configuration customization (optional)
-**********************************************
+      sudo systemctl enable --now docker.service
 
-|CL| is a stateless system that looks for user-defined package configuration files in the :file:`/etc/<package-name>` directory to be used as default. If user-defined files are not found, |CL| uses the distribution-provided configuration files for each package.
+#. Configure kubelet to use the |CL| directory for cni-plugins and reload the
+   service.
 
-If you customize any of the default package configuration files, you **must** store the customized files in the :file:`/etc/` directory. If you edit any of the distribution-provided default files, your changes will be lost in the next system update as the default files will be overwritten with the updated files.
+   .. code-block:: bash
 
-For example, to customize CRI-O configuration in your system, run the following commands:
+      sudo mkdir -p  /etc/systemd/system/kubelet.service.d/
 
-.. code-block:: bash
+      cat << EOF | sudo tee  /etc/systemd/system/kubelet.service.d/0-cni.conf
+      [Service]                                                 
+      Environment="KUBELET_EXTRA_ARGS=--cni-bin-dir=/usr/libexec/cni"
+      EOF
 
-   sudo mkdir /etc/crio
-   sudo cp /usr/share/defaults/crio/crio.conf /etc/crio/
-   sudo $EDITOR /etc/crio/crio.conf
+   .. code:: bash
 
-Learn more about :ref:`stateless` in |CL|.
+      sudo systemctl daemon-reload
+
+
+#. Run the kubeadm command to initialize the master node:
+
+   .. important:: 
+
+      You may need to add additional parameters to the command below,
+      depending the pod network addon in use. 
+      
+      In this example, the :command:`--pod-network-cidr 10.244.0.0/16`
+      parameter is to use *flannel* as the pod networking. See `Choose a pod
+      network add-on`_ for more information.
+
+   .. code:: bash
+
+      sudo kubeadm init \
+      --pod-network-cidr 10.244.0.0/16 
+
+
+#. Once the cluster initialization is complete, continue reading about how to
+   `Use your cluster`_.
+   
+
+
+Add the Kata runtime to Kubernetes
+-----------------------------------
+
+For information on Kata as a container runtime, see `What is Kata
+Containers?`_. Using Kata Containers is optional.
+
+You can use *kata-deploy* to install all the necessary parts of Kata
+Containers after you have a Kubernetes cluster running with one of the CRI's
+using the default runc runtime. Follow the steps in the Kubernetes quick start
+section of the  `kata-containers GitHub README
+<https://github.com/kata-containers/packaging/tree/master/kata-deploy#kubernetes-quick-start>`_
+to install Kata.
```

Comment:
Kata runtime is already part of `cloud-native-basic` bundle so this is not needed. Only the part that adds ? 
`RuntimeClass` settings is needed.